SECURITY.md

```diff
@@ -0,0 +1,24 @@
+<!--
+******************************************************************************
+* Copyright 2023 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/-->
+

```

Comment:
@aepanchi  @napetrov  @rozhukov - can we add information on SDL process followed and tools beings used?  Do we have good examples of this being done by other projects?

Reply:
Please use approved wording for security.md, for example from: https://github.com/intel/linux-sgx/blob/master/SECURITY.md

I can't say it's the "best industry example" though, but it's a bar minimum.

Reply:
Done.

Reply:
But this just focused on Reporting vulnerabilities. Do we have anything that would explain intel's SDL process?